pkgs/by-name/nu/nuv/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, pkgs
+, callPackage
+, fetchFromGitHub
+, fetchurl
+, buildGoModule
+, makeWrapper
+, breakpointHook
+, jq
+, curl
+, kubectl
+, eksctl
+, kind
+, k3sup
+, coreutils
+}:
+
+let
+  branch = "3.0.0";
+  version = "3.0.1-beta.2405292059";
+in
+buildGoModule rec {
```

Comment:
```suggestion
buildGoModule {
```

Is this rec needed?

Reply:
Hello, that "rec" is needed because the passthru tests inherit pname version;

Reply:
What about putting them in the let-in already in use instead?

Reply:
Done. Let me know if now is ok!